doc/how_to/make_a_sorting.rst

```diff
@@ -0,0 +1,89 @@
+    phy_sorting = read_phy('path/to/folder')
+
+This :code:`Sorting` contains important information about your spike trains including
+the spike times (i.e. when the neurons were actually firing) the unit labels (i.e.
+who the spikes belong to. Also called cluster ids by some sorters), the unit ids (the unique
+set of unit labels) and the sampling_frequency. To make your own :code:`Sorting` object you can
+use :code:`NumpySorting`. It is important to note that in SpikeiInterface spike trains are handled internally in samples/frames rather than in seconds and we use the sampling frequency to ...
+are typically stored in samples/frames rather than in seconds. So you should input the times
+in samples/frames. The sampling_frequency allows for easily switching between samples and seconds.
+
+There are 3 options (along with making a NumpySorting from another sorting which will not be covered here):
+
+With lists of spike trains and spike labels
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+In this case we need a list or array (or lists of lists for multisegment) of spike times,
+unit labels, sampling_frequency and optional unit_ids if you want specific labels to be
+used (in this case we only create the :code:`Sorting` based on the requested unit_ids).
+
+.. code-block:: python
+
+    from spikeinterface.core import NumpySorting
+
+    # in this case we are making a monosegment sorting
+    my_sorting = NumpySorting.from_times_labels(times_list = [1,2,3,4],
+                                                labels_list = [0,1,0,1],
+                                                sampling_frequency = 30_000.0
+                                                )
+
+
+With a unit dictionary
+^^^^^^^^^^^^^^^^^^^^^^
+
+We can also use a dictionary where each unit is a key and its spike times are values.
+This is entered as either a list of dicts with each dict being a segment or as a single
+dict for monosegment. We still need to separately specify the sampling_frequency
+
+.. code-block:: python
+
+    from spikeinterface.core import NumpySorting
+
+    my_sorting = NumpySorting.from_unit_dict(units_dict_list={'0': [1,3],
+                                                              '1': [2,4]
+                                                              },
+                                             sampling_frequency=30_000.0
+                                           )
+
+
+With Neo SpikeTrains
+^^^^^^^^^^^^^^^^^^^^
+
+Finally since SpikeInterface is tightly integrated with the Neo project you can create
+a sorting from :code:`Neo.SpikeTrain` objects. See Neo documentation for more information on
+using :code:`Neo.SpikeTrain`'s.
+
+.. code-block:: python
+
+    from spikeinterface.core import NumpySorting
+
+    # neo_spiketrain is a Neo spiketrain object
+    my_sorting = NumpySorting.from_neo_spiketrain_list(neo_spiketrain,
+                                                       sampling_frequency=30_000.0)
```

Comment:
I'd add a line, just to wrap up the guide. Something like:

"
Now that you've created a Sorting object, you can combine it with a recording to make a :ref:`Sorting Analyzer <sphx_glr_tutorials_core_plot_4_sorting_analyzer.py>`, or start visualising by using the :py:func:`~spikeinterface.widgets.plot_crosscorrelograms` function.
"

But up to you!